templates/download/xilinx/index.html

```diff
@@ -32,77 +32,298 @@ <h1>Install Ubuntu on Xilinx</h1>
+              </tr>
+              <tr>
+                <th><a href="https://people.canonical.com/~platform/images/xilinx/rootfs.tar.gz">rootfs.tar.gz</a></th>
+                <td colspan="2">Raw Root filesystem</td>
+              </tr>
+              <tr>
+                <th><a href="https://people.canonical.com/~platform/images/xilinx/system-boot.tar.gz">system-boot.tar.gz</a></th>
+                <td colspan="2">FAT Partition Contents</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <div class="u-fixed-width">
+          <p>If you have any questions about the platform or would like information about our certification program, please contact us.</p>
+          <p><a href="/download/contact-us" class="js-invoke-modal">Get in touch&nbsp;&rsaquo;</a></p>
+        </div>
+      </section>
     </div>
-  </div>
-</section>
 
-<section class="p-strip">
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h3 class="p-heading--4">First time installing Ubuntu on Xilinx?</h3>
-      <p>Please see the <a class="p-link--external" href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/2037317633/Getting+Started+with+Certified+Ubuntu+20.04+LTS+for+Xilinx+Devices?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=download&utm_term=desktop_image">Xilinx Getting Started Guide</a> for more information.</p>
-      <p><strong>Xilinx Ubuntu Software Development Kit</strong></p>
-      <p>Ubuntu Developers targeting Xilinx Zynq UltraScale+ MPSoC devices may also want to download the following files:</p>
+    <div tabindex="0" role="tabpanel" id="kira-kv260-vision-ai-tab" aria-labelledby="kira-kv260-vision-ai" hidden="true">
+      <section class="p-strip--light">
+        <div class="row u-vertically-center">
+          <div class="col-6">
+            <h2>Download Ubuntu Desktop</h2>
+            <h3 class="p-heading--4">Ubuntu Desktop {{ releases.lts.full_version }} LTS</h3>
+            <p class="u-sv1">The version of Ubuntu with up to 10 years of long term support, until April 2030.</p>
+            <p>
+              <a class="p-button--positive" href="https://people.canonical.com/~platform/images/xilinx/kria/iot-kria-classic-desktop-2004-x03-20211110-98.img.xz">Download 64-bit</a>
+            </p>
+            <p>Works on:</p>
+            <ul class="p-list">
+              <li class="p-list__item is-ticked">Xilinx Kria KV260 Vision AI Starter Kit </li>
+            </ul>
+          </div>
+          <div class="col-6 u-align--center u-hide--small">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/4ede62b5-Xilinx-board.png",
+              alt="Xilinx Kria KV260 Vision AI circuit board",
+              width="380",
+              height="335",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+        </div>
+      </section>
+
+      <section class="p-strip">
```

Comment:
Please remove the padding from the top of this strip

Reply:
I think I should have dealt with the spacing issue by removing the nested p-strips